qa/tasks/cephfs/test_failover.py

```diff
@@ -103,8 +103,6 @@ def test_join_fs_vanilla(self):
         """
         That a vanilla standby is preferred over others with mds_join_fs set to another fs.
         """
-        # After Octopus is EOL, we can remove this setting:
```

Comment:
I think this is okay. We don't run these tests against an Octopus cluster with master QA.